articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,496 @@
+
+    ![Azure site recovery installation 8](images/azs-siterecovery-installer-step8.png)
+
+10. In **Network Selection**, specify the listener (network adapter and SSL port) on which the configuration server sends and receives replication data. Port 9443 is the default port used for sending and receiving replication traffic, but you can modify this port number to suit your environment's requirements. In addition to the port 9443, we also open port 443, which is used by a web server to orchestrate replication operations. Do not use port 443 for sending or receiving replication traffic.
+
+    ![Azure site recovery installation 9](images/azs-siterecovery-installer-step9.png)
+
+11. In **Summary**, review the information and click Install. When installation finishes, a passphrase is generated. You will need this when you enable replication, so copy it and keep it in a secure location.
+
+    ![Azure site recovery installation 10](images/azs-siterecovery-installer-step10.png)
+
+    ![Azure site recovery installation 11](images/azs-siterecovery-installer-step11.png)
+
+After registration finishes, the server should become an option when selecting **Configuration Server**.
+
+## Step 4. Set up target environment
+
+In Public Azure, select and verify target resources.
+
+1. In **Prepare infrastructure** > **Target**, select the Azure subscription you want to use.
+
+2. Specify the target deployment model.
+
+3. Site Recovery checks that you have one or more compatible Azure storage accounts and networks. If it doesn't find them, you need to create at least one storage account and virtual network, in order to complete the wizard.
+
+4. Select **OK**.
+
+    ![Azure site frecovery infrastructure 2](images/azs-browser-recovery-infrastructure-configuration3.png)
+
+## Step 5: Enable replication
+
+1. Select the final tab **Replication Settings**.
+
+2. Select **Create and Associate** and specify a policy name.
+
+3. In **RPO threshold**, specify the recovery point objective (RPO) limit.
+
+    - Recovery points for replicated data are created in accordance with the time set.
+
+    - This setting does not affect replication, which is continuous. It simply issues an alert if the threshold limit is reached without a recovery point being created.
+
+4. In **Recovery point retention**, specify how long each recovery point is kept. Replicated VMs can be recovered to any point in the specified time window.
+
+5. In **App-consistent snapshot frequency**, specify how often application-consistent snapshots are created.
+
+    - A app-consistent snapshot is a point-in-time snapshot of the app data inside the VM.
+
+    - Volume Shadow Copy Service (VSS) ensures that apps on the VM are in a consistent state when the snapshot is taken.
+
+6. Select **OK** to create the policy.
+
+    ![Azure site frecovery infrastructure 3](images/azs-browser-recovery-infrastructure-configuration2.png)
```

Comment:
"frecovery" and think this should be 2, not 3

Reply:
done